CustomTracks/Backgrounds/CustomBackground.cs

```diff
@@ -166,6 +166,25 @@ public override void SetUpBackground(BGController controller, GameObject bg)
                 ? gameController.textureindex
                 : (int)trombonePlaceholder.TromboneSkin);
             
+            
+            if (trombonePlaceholder.TrombonerOutfit == TrombonerOutfit.Christmas) {
+                Material[] materials = tromboner.controller.bodymesh.materials;
+                materials[0] = tromboner.controller.costume_alt;
+                tromboner.controller.bodymesh.materials = materials;
+            }
+            
+            var chosen_hat = trombonePlaceholder.TromboneHat == TromboneHat.DoNotOverride
+            ? GlobalVariables.chosen_hat : (int)trombonePlaceholder.TromboneHat;
+
+            if (chosen_hat > 0)
+            {
+                GameObject hat = Object.Instantiate(controller.gamecontroller.hats[chosen_hat - 1], 
```

Comment:
we have `GameController` bound to a local variable `gameController` in this scope

Reply:
sorted 👍